### DISCUSSION TITLE
csrf_session_id

### DISCUSSION THREAD

**Discussion by chengazhen** [Johnserf-Seed/TikTokDownload#553] [Q&A]
想请教一下怎么通过接口获取 session_id， 我现在基于此项目通过electron来实现TikTokDownload-web，cookie里面有一个session_id，其他参数都可以通过扫码登录来后去，唯独这个id没有找到如何获取？

**Comment by Johnserf-Seed** [comment]
你不是也说了cookie里面有一个session_id，你最后的这个id没有找到是什么意思

**Comment by chengazhen** [comment]
不好意思我没有表达清楚，就是我执行了一遍扫码登录的流程获取到了除sessionID以外的参数，想请教一下这个cookie的获取方式

**Comment by chengazhen** [comment]
这是我的代码仓库, 用 electron 实现了当前仓库的扫码功能; 我想通过这个方式来获取cookie并且应用到 TikTokDownload-web 端, 现在卡在不知道怎么获取sessionid;
https://github.com/Journey98/TikTokWeb-electron/blob/refactory/src/main/utils/login.js#L79
![image](https://github.com/Johnserf-Seed/TikTokDownload/assets/52627267/99241e88-fbf6-4a73-8ce6-3ddd656980ad)
我也查看了当前仓库的部分代码, 发现python实现的功能貌似只用到了扫码登录最终获取的一个cookie, 并没有像web端使用多个cookie, 

最后问一下有相关的交流群吗?
